### ISSUE TITLE
Compatibility with Joblib 1.2(.0)

### ISSUE THREAD

**Issue by vruusmann** [jpmml/jpmml-python#19]
I'm sorry for bumping this old and closed issue up, but strangely I got the same error now, which has never been occurred to me before. To make sure, I ran the LightGBM example from your blog entry. It resulted the same error.

```
Standard output is empty
Standard error:
Exception in thread "main" net.razorvine.pickle.InvalidOpcodeException: invalid pickle opcode: 0
	at net.razorvine.pickle.Unpickler.dispatch(Unpickler.java:366)
	at org.jpmml.python.CustomUnpickler.dispatch(CustomUnpickler.java:31)
	at org.jpmml.python.PickleUtil$1.dispatch(PickleUtil.java:64)
	at net.razorvine.pickle.Unpickler.load(Unpickler.java:109)
	at org.jpmml.python.PickleUtil.unpickle(PickleUtil.java:85)
	at com.sklearn2pmml.Main.run(Main.java:78)
	at com.sklearn2pmml.Main.main(Main.java:66)
```

Here is my environment:
```
python-3.10.5.amd64

sklearn2pmml         0.86.3
scikit-learn         1.1.2
pandas               1.5.0
sklearn-pandas       2.2.0
joblib               1.2.0
numpy                1.23.3

java version "1.8.0_141"
Java(TM) SE Runtime Environment (build 1.8.0_141-b15)
Java HotSpot(TM) 64-Bit Server VM (build 25.141-b15, mixed mode)
```

Is there anything that I can do to investigate further? Thank you

**EDIT:**
Reverting joblib to 1.1.0 solves the problem.

_Originally posted by @denmase in https://github.com/jpmml/sklearn2pmml/issues/8#issuecomment-1260263014_

**Comment by agude-sq** [comment]
We were getting the same error with `joblib == 1.2.0` and again downgrading to `1.1.0` solved it.

**Comment by vruusmann** [comment]
The fix is included in SkLearn2PMML version 0.87.0 (and newer).

The unexpected zero byte inside Joblib 1.2.0 pickles is related to "padding for efficient Numpy array memory-mapping functionality", as detailed in the closing commit (referenced above).
